docs/Concepts/Client-Libraries.md

```diff
@@ -0,0 +1,13 @@
+description: Hyperledger Besu client libraries 
+<!--- END of page meta data -->
+
+# Client Libraries
+
+Client libraries such as web3.js, web3j, or ethereumj are used by Dapps to forward JSON-RPC requests to Hyperledger Besu. Any
+client library that implements core Ethereum RPC methods can be used with Besu.
+
+The web3.js-eea library can be used with Besu for [privacy features](../Concepts/Privacy/Privacy-Overview.md).
```

Comment:
The web3.js-eea library can be used -> Use the web3.js-eea library with Besu .... 
Edit to make active. 

Add a link on web3.js-eea library to /HowTo/Interact/Client-Libraries/eeajs/

Reply:
Done.